deploy/bundle/metadata/annotations.yaml

```diff
@@ -0,0 +1,14 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: devworkspace-operator
+  operators.operatorframework.io.bundle.channels.v1: alpha
```

Comment:
IIRC someone made a comment a while ago about us using the alpha channel on WTO. Should we use something like main?

Reply:
Apparently OLM has some recommendations on their website: https://olm.operatorframework.io/docs/best-practices/channel-naming/#naming

Reply:
Good point -- WDYT about `fast` for now? We're certainly not `stable` :smile:

Reply:
`fast` sounds about right for now :laughing: